session/src/test/java/org/apache/iotdb/session/IoTDBSessionSimpleIT.java

```diff
@@ -665,7 +665,7 @@ private void checkResult(Session session)
           "root.sg.d1.s2"
         });
     Assert.assertArrayEquals(
-        dataSet.getColumnTypes().toArray(new TSDataType[0]),
+        dataSet.getColumnTypes().toArray(new String[0]),
         new TSDataType[] {
           TSDataType.INT64,
           TSDataType.INT64,
```

Comment:
Hi, I wonder whether this change is correct, since the class is `TSDataType` and `String` separately.
I got this failure when I run tests in `IoTDBSessionSimpleIT.java`:
![20431622707611_ pic](https://user-images.githubusercontent.com/19167280/120614520-e7981700-c489-11eb-89ed-11b61a552912.jpg)

Reply:
After I changed the codes like this:
```
Assert.assertArrayEquals(
        dataSet.getColumnTypes().toArray(new String[0]),
        new String[] {
          TSDataType.INT64.toString(),
          TSDataType.INT64.toString(),
          TSDataType.FLOAT.toString(),
          TSDataType.BOOLEAN.toString(),
          TSDataType.INT32.toString(),
          TSDataType.INT32.toString()
        });
```
the tests would be passed